test/cp.js

```diff
@@ -797,7 +797,7 @@ test('should not attempt to copy fifos via symlinks in directories', t => {
 test('should copy fifos directly', t => {
   try {
     shell.exec(`mkfifo ${t.context.tmp}/fifo`);
-    shell.exec(`echo -n test1 > ${t.context.tmp}/fifo`, { async: true });
+    shell.exec(`printf test1 > ${t.context.tmp}/fifo`, { async: true });
```

Comment:
I'd prefer to see `shx echo -n test1` here. Appveyor may have the command, but it may not be consistent with unix `printf`, and I don't think it's available by default on Windows systems.